packages/uikit-react-native/src/contexts/ReactionCtx.tsx

```diff
@@ -7,6 +7,7 @@ import { ReactionBottomSheets } from '../components/ReactionBottomSheets';
 import { LocalizationContext } from '../contexts/LocalizationCtx';
 import { SendbirdChatContext } from '../contexts/SendbirdChatCtx';
 import { UserProfileContext } from '../contexts/UserProfileCtx';
+import { useReaction } from '../hooks/useContext';
```

Comment:
gist of changes:
The ReactionProvider provides context but also renders the bottom sheet modals. All I did was separate out the two so that I could wrap context around the Provider render prop I introduce in this PR, which can then wrap the bottom sheet modals.